telegram_bot_calendar/base.py

```diff
@@ -78,18 +78,18 @@ def __init__(self, calendar_id=0, current_date=None, additional_buttons=None, lo
     @staticmethod
     def func(calendar_id=0):
         def inn(callback):
-            start = CB_CALENDAR + "_" + str(calendar_id)
+            start = CB_CALENDAR + "#" + str(calendar_id)
             return callback.data.startswith(start)
 
         return inn
 
-    def build(self):
+    def build(self, **kwargs):
```

Comment:
I think it would be nice to throw an exception here if the size of the callback data is more than 64 bytes. 
Previously, it was guaranteed that no telegram API exception occurs.
Please, see: https://core.telegram.org/bots/api#inlinekeyboardmarkup